nixos/modules/services/development/jupyter/default.nix

```diff
@@ -37,6 +38,27 @@ in {
       '';
     };
 
+    package = mkOption {
+      type = types.package;
+      # NOTE: We don't use top-level jupyter because we don't
+      # want to pass in JUPYTER_PATH but use .environment instead,
+      # saving a rebuild.
+      default = pkgs.python3.pkgs.notebook;
+      description = ''
+        Jupyter package to use.
+      '';
+    };
+
+    useJupyterLab = mkOption {
```

Comment:
This is not sufficiently flexible. When a 3rd tool comes, what then? Better to configure the executable name.